x/ibc/03-connection/keeper/verify.go

```diff
@@ -32,19 +32,27 @@ func (k Keeper) VerifyClientConsensusState(
 // specified connection end stored on the target machine.
 func (k Keeper) VerifyConnectionState(
 	ctx sdk.Context,
+	connection types.ConnectionEnd,
```

Comment:
cc: @cwgoes I noticed the spec wasn't updated and consistent in all the ICS'. Can you double-check this logic is correct?

Reply:
It looks correct to me. What was inconsistent in the spec? I found one mistake - https://github.com/cosmos/ics/issues/371 - but if there are more please let me know & I will fix them.